### ISSUE TITLE
Re-Publish English Content to API

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord-dev/door43.org#348]
* [x] tN - version 5
* [x] tW - version 5
* [x] tQ - version 5
* [x] ULB - version 8
* [x] UDB - version 8

Blocked by #332

**Comment by phillip-hopper** [comment]
Ran into a couple encoding issues with tW. Someone ran some scripts to modify files and in the process changed some to `iso-8859-1` encoding and other to `us-ascii` encoding. I was able to convert the `iso-8859-1` files to `utf-8` using a shell script, but not the `us-ascii` because it is considered a subset of `utf-8` and therefore there is no need to convert. However, there are apparently unicode characters in the ascii files, which causes errors. I'm having to convert them one at a time when I run into an exception. This is a screen shot from Gogs that illustrates the problem:

![image](https://cloud.githubusercontent.com/assets/5262964/20909925/78c53b66-bb2c-11e6-8829-0b645887e37e.png)
